feature-onboarding-impl/src/main/java/jp/co/soramitsu/onboarding/impl/welcome/WelcomeViewModel.kt

```diff
@@ -72,14 +81,46 @@ class WelcomeViewModel @Inject constructor(
         }
 
         viewModelScope.launch {
-            onboardingInteractor.getConfig()
+            val remoteOnboardingConfig: OnboardingConfig? = onboardingInteractor.getConfig()
```

Comment:
It would be better to create a method in OnboardingInteractor

suspend fun getConfig(): OnboardingConfig? {
// 1. load config
// 2. getAppVersionSupportedConfig()
// 3. return already filtered config or null
}